lithops/serverless/backends/ibm_cf/ibm_cf.py

```diff
@@ -67,41 +65,19 @@ def __init__(self, ibm_cf_config, storage_config):
                                              namespace=self.namespace,
                                              auth=auth,
                                              user_agent=self.user_agent)
+
         elif self.iam_api_key:
-            token_manager = DefaultTokenManager(api_key_id=self.iam_api_key)
-            token_filename = os.path.join(CACHE_DIR, 'ibm_cf', 'iam_token')
-
-            if 'token' in self.ibm_cf_config:
-                logger.debug("Using IBM IAM API Key - Reusing Token from config")
-                token_manager._token = self.ibm_cf_config['token']
-                token_manager._expiry_time = datetime.strptime(self.ibm_cf_config['token_expiry_time'],
-                                                               '%Y-%m-%d %H:%M:%S.%f%z')
-                token_minutes_diff = int((token_manager._expiry_time - datetime.now(timezone.utc)).total_seconds() / 60.0)
-                logger.debug("Token expiry time: {} - Minutes left: {}".format(token_manager._expiry_time, token_minutes_diff))
-
-            elif os.path.exists(token_filename):
-                logger.debug("Using IBM IAM API Key - Reusing Token from local cache")
-                token_data = load_yaml_config(token_filename)
-                token_manager._token = token_data['token']
-                token_manager._expiry_time = datetime.strptime(token_data['token_expiry_time'],
-                                                               '%Y-%m-%d %H:%M:%S.%f%z')
-                token_minutes_diff = int((token_manager._expiry_time - datetime.now(timezone.utc)).total_seconds() / 60.0)
-                logger.debug("Token expiry time: {} - Minutes left: {}".format(token_manager._expiry_time, token_minutes_diff))
-
-            if (token_manager._is_expired() or token_minutes_diff < 11) and not is_lithops_worker():
```

Comment:
@JosepSampe why this code removed?

Reply:
Because of what I said in previous comment.